modules/mobile/spec/workers/pre_cache_appointments_job_spec.rb

```diff
@@ -31,64 +29,85 @@
 
     after { Timecop.return }
 
-    context 'with no errors' do
-      it 'caches the expected appointments' do
+    it 'caches the expected appointments' do
+      VCR.use_cassette('appointments/get_facilities', match_requests_on: %i[method uri]) do
+        VCR.use_cassette('appointments/get_cc_appointments_default', match_requests_on: %i[method uri]) do
+          VCR.use_cassette('appointments/get_appointments_default', match_requests_on: %i[method uri]) do
```

Comment:
some of these specs are still using v0 appointments. In the current draft PR for ripping out v0 appointments, we're removing all v0 specs with future tickets to make v2 versions of them. so these will be removed soon. Could change them to v2 now while your working with them or do it later.